docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+    wildduck:
+        network_mode: host
+        build: .
+        ports:
+            - "8080:8080"
```

Comment:
We need to expose more than just 8080. You'll need 143, 110, 993 and 995 for IMAP and POP3 and Haraka will have 465 and 587 for SMTP.

Reply:
465 and 587 is SMTP, for haraka. Not for Wildduck.